CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## 20220217-0c4e97a
```

Comment:
See description for my thinking around this choice.

Reply:
Updated description with a change in https://github.com/Intellection/docker-redis-roaring/pull/1/commits/3da62ab3e8a0da7761815f06d5db26d32a797751.

Reply:
Noted. Fine by me.